server/cluster.go

```diff
@@ -325,10 +325,12 @@ func (s *Server) bootstrapCluster(clusterID uint64, req *pdpb.BootstrapRequest)
 	ops = append(ops, clientv3.OpPut(regionSearchPath, string(regionValue)))
 
 	bootstrapCmp := clientv3.Compare(clientv3.CreatedRevision(clusterRootPath), "=", 0)
-	resp, err := s.client.Txn(context.TODO()).
+	ctx, cancel := context.WithTimeout(context.Background(), requestTimeout)
+	resp, err := s.client.Txn(ctx).
```

Comment:
It is hard to handle it even we don't use request timeout, because we may still not know whether success or fail with other network errors.
I think we should handle these txns one by one later.